src/foursquare-lunch.coffee

```diff
@@ -14,6 +14,12 @@
 # Author:
 #   stephenyeargin
 
+
+# TODO: error if FOURSQUARE variables not set.
```

Comment:
I'd rather these be :octocat: [issues](https://github.com/stephenyeargin/hubot-foursquare-lunch/issues/new) rather than inline in the code. That keeps them in a place where folks are more likely to see them and perhaps contribute PRs.